besu/src/main/scripts/unixStartScript.txt

```diff
@@ -182,15 +182,15 @@ APP_ARGS=`save "\$@"`
 # Collect all arguments for the java command, following the shell quoting and substitution rules
 eval set -- \$DEFAULT_JVM_OPTS \$JAVA_OPTS \$${optsEnvironmentVar} <% if ( appNameSystemProperty ) { %>"\"-D${appNameSystemProperty}=\$APP_BASE_NAME\"" <% } %>-classpath "\"\$CLASSPATH\"" <% if ( mainClassName.startsWith('--module ') ) { %>--module-path "\"\$MODULE_PATH\"" <% } %>${mainClassName} "\$APP_ARGS"
 
-unset BESU_USING_JEMALLOC
+#unset BESU_USING_JEMALLOC
 if [ "\$darwin" = "false" -a "\$msys" = "false" ]; then
   # check if jemalloc is available
   TEST_JEMALLOC=\$(LD_PRELOAD=libjemalloc.so sh -c true 2>&1)
 
   # if jemalloc is available the output is empty, otherwise the output has an error line
   if [ -z "\$TEST_JEMALLOC" ]; then
     export LD_PRELOAD=libjemalloc.so
-    export BESU_USING_JEMALLOC=true
+#    export BESU_USING_JEMALLOC=true
```

Comment:
how would `BESU_USING_JEMALLOC` be set to true if not here?

Reply:
BESU_USING_JEMALLOC is supposed to be a environment variable set outside Besu, at the OS or container level. If it's not set it will load Besu if installed, like is the behavior today.